File: samples/ExampleWebFormsApplication/Site.Master.cs
using System;

namespace ExampleWebFormsApplication
{
    public partial class SiteMaster : System.Web.UI.MasterPage
    {
        /// <summary>Exposed to content pages through the typed <c>Master</c> property that the generator creates from <c>&lt;%@ MasterType %&gt;</c>.</summary>
        public string Heading
        {
            get { return lblHeading.Text; }
            set { lblHeading.Text = value; }
        }

        protected void Page_Load(object sender, EventArgs e)
        {
            litFooter.Text = "Rendered " + DateTime.Now.ToString("u");
            Head1.Title = Heading;
        }
    }
}

File: samples/ExampleWebFormsApplicationVB/Site.Master.vb
Public Class SiteMaster
    Inherits System.Web.UI.MasterPage

    ''' <summary>Exposed to content pages through the typed Master property the generator creates from &lt;%@ MasterType %&gt;.</summary>
    Public Property Heading As String
        Get
            Return lblHeading.Text
        End Get
        Set(value As String)
            lblHeading.Text = value
        End Set
    End Property

    Protected Sub Page_Load(sender As Object, e As EventArgs) Handles Me.Load
        Head1.Title = Heading
    End Sub
End Class
